lib/adal-angular.js

```diff
@@ -320,7 +320,7 @@ if (typeof module !== 'undefined' && module.exports) {
                             if (authService.loginInProgress()) {
                                 authService.info('login already start.');
                                 return $q.reject();
-                            } else if (authService.config && isEndpoint) {
+                            } else if (authService.config && (!authService.config.endpoints || isEndpoint)) {
```

Comment:
@sarmut `getResourceForEndpoint` at line 297 does an additional check matching the uri against app's client id. In that case, isEndpoint would be false, because `client_id` is not part of config.endpoints. app's client_is is the default endpoint.